modules/identity-provider-registering-github.adoc

```diff
@@ -17,19 +17,20 @@ https://github.com/settings/applications/new[Register a new OAuth application].
 ** For GitHub Enterprise, go to your GitHub Enterprise home page and then click
 *Settings -> Developer settings -> Register a new application*.
 . Enter an application name, for example `My OpenShift Install`.
-. Enter a homepage URL, such as `https://myapiserver.com:8443`.
+. Enter a homepage URL, such as 
+`https://console-openshift-console.apps.example-openshift-cluster.com/`.
 . Optionally, enter an application description.
 . Enter the authorization callback URL, where the end of the URL contains the
 identity provider `name`:
 +
 ----
-<apiserver>/oauth2callback/<identityProviderName>
+https://openshift-authentication-openshift-authentication.apps.<clusterDomain>/oauth2callback/<identityProviderName>/
```

Comment:
Should be https://oauth-openshift.apps.<cluster-name>.<cluster-domain>/oauth2callback/<idp-provider-name>

Reply:
Corrected to use the given format.